web/js/general.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function () {
+    $('html').removeClass('no-js');
+
+    // To prevent duplicates, use the first locale options as a reference for building buttons
+    var localeOptions = $('.locale-switch:first option');
+
+    for (var i = 0; i < localeOptions.length; i++) {
+        var localeDataAttribute = 'data-locale="'+ localeOptions[i].value +'"';
+
+        var localeOptionClassAttribute = '';
+        if (localeOptions[i].selected) {
+            localeOptionClassAttribute = 'class="active"';
+        }
+
+        var localeOptionButton =
+            '<button ' + localeOptionClassAttribute + ' ' + localeDataAttribute + '>'
+            + localeOptions[i].text
+            + '</button>';
```

Comment:
Noooooooooooo! These string concatenations should trigger your XSS senses! Use jQuery. *(Hint: `$('<button>')`)*